Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,14 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const inHours = tasks.map(hour => hour['duration']/60);
+console.log(inHours);
```

Comment:
Do not leave console.log that you used for debug purposes when you submit your final work in a PR.